data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1098,7 +1098,7 @@ spec:
                     description: ClusterOSImage is either a URL with `http(s)` or `file` scheme to override the default OS image for cluster nodes, or an existing Glance image name.
                     type: string
                   computeFlavor:
-                    description: FlavorName is the name of the compute flavor to use for instances in this cluster.
+                    description: 'DeprecatedFlavorName is the name of the compute flavor to use for instances in this cluster. Deprecated: use FlavorName in DefaultMachinePlatform to define default compute flavor.'
                     type: string
                   defaultMachinePlatform:
```

Comment:
I just noticed that defaultMachinePlatform isn't documented in docs/user/openstack/customization.md. We'll need to fix that too.

Reply:
done!